app_dart/lib/src/service/branch_service.dart

```diff
@@ -134,12 +134,14 @@ class BranchService {
     required gh.RepositorySlug slug,
     required String branchName,
   }) async {
-    return (await githubService.getFileContent(
-      slug,
-      'bin/internal/release-candidate-branch.version',
-      ref: branchName,
-    ))
-        .trim();
+    return githubService
+        .getFileContent(
+          slug,
+          'bin/internal/release-candidate-branch.version',
+          ref: branchName,
+        )
+        .then((String value) => value.trim())
+        .onError((e, _) => '');
```

Comment:
Can we add a TODO here with the follow up issue linked. That may be easier than logging.

Reply:
Sure, added a comment on why we return empty string. Yeah the expectation is that this would be the PR so that no follow up fix is needed.